log/src/main/java/kcl/teamIndexZero/traffic/log/Log_Config.java

```diff
@@ -1,6 +1,6 @@
 package kcl.teamIndexZero.traffic.log;
 
-import kcl.teamIndexZero.traffic.log.fileIO.FileInput;
+import kcl.teamIndexZero.traffic.log.FileIO.FileInput;
```

Comment:
What are you referring to? The double import statements? What is more surprising is that I didn't even touch the log files so don't know how did that end up there.